src/sdk/accountspace_nb.js

```diff
@@ -41,478 +36,140 @@ class AccountSpaceNB {
 
     async create_user(params, account_sdk) {
 
-        const action = IAM_ACTIONS.CREATE_USER;
         const requesting_account = system_store.get_account_by_email(account_sdk.requesting_account.email);
```

Comment:
I think you can avoid this call to the DB, in `accountspace_fs` it was `const requesting_account = account_sdk.requesting_account;`